pyscriptjs/src/components/pyconfig.ts

```diff
@@ -28,24 +42,51 @@ export class PyConfig extends BaseEvalElement {
     }
 
     connectedCallback() {
-        this.code = this.innerHTML;
-        this.innerHTML = '';
+        let loadedValues: object = {};
+        let srcConfig: object = {};
+        let inlineConfig: object = {};
 
-        const loadedValues = jsyaml.load(this.code);
-        if (loadedValues === undefined) {
-            this.values = {
-                autoclose_loader: true,
-                runtimes: [DEFAULT_RUNTIME_CONFIG]
-            };
-        } else {
-            // eslint-disable-next-line
-            // @ts-ignore
-            this.values = loadedValues;
+        // load config from src and inline
+        if (this.hasAttribute('src') && this.innerHTML!=='')
+        {
+            srcConfig = JSON.parse(readTextFromPath(this.getAttribute('src')));
```

Comment:
JSON.parse needs error handling for SyntaxError. How do we convey this error to developers?
```javascript
try{
  const result = JSON.parse({});
} catch (err) {
  // SyntaxError
  console.log(err.message);
}
```